dashboard/src/actions/billing.ts

```diff
@@ -0,0 +1,18 @@
+'use server';
+
+import { withUserAuth } from '@/auth/auth-actions';
+import { getUserBillingStats, getUserBillingHistory } from '@/services/billing';
+import { type UserBillingData } from '@/entities/billing';
+
+export const getUserBillingData = withUserAuth(async (userId: string): Promise<UserBillingData> => {
+  return await getUserBillingStats(userId);
```

Comment:
Quibble: No need to await
```suggestion
  return getUserBillingStats(userId);
```